dauphine-open-data/src/main/java/io/github/oliviercailloux/y2018/opendata/resource/PersonResource.java

```diff
@@ -1,15 +1,48 @@
 package io.github.oliviercailloux.y2018.opendata.resource;
 
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
 import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
 
+import ezvcard.VCard;
+import ezvcard.Ezvcard;
+
+import io.github.oliviercailloux.y2018.opendata.dao.EntityAlreadyExistsDaoException;
 import io.github.oliviercailloux.y2018.opendata.dao.PersonDao;
 import io.github.oliviercailloux.y2018.opendata.entity.Person;
+import io.github.oliviercailloux.y2018.opendata.mapper.VcardImport;
 
 @Path("person")
 public class PersonResource extends AbstractResource<Person, PersonDao> {
 
 	public PersonResource() {
 		super("Person", "person");
 	}
-
-}
+	
+	/**
+	 * @param lastName of the person
+	 * @param firstName of the person
+	 * @param ine of the person
+	 * @param office of the person
+	 * @param phoneNumber of the person
+	 * @param training of the person
+	 * @param fax of the person
+	 * @param mail of the person
+	 * 
+	 * We use VcardImport to import VCard information 
+	 * then we persist them using PersonDao
+	 * 
+	 * @return person
+	 */
+	@POST
+	@Path("importvcard")
+	@Consumes("text/plain")
+	public Response importVCard(String vCardString) throws EntityAlreadyExistsDaoException {
```

Comment:
Préciser en commentaire SVP quelles versions de vcard sont acceptées ; et chercher quel type mime utiliser.

Reply:
C'est corrigé

Reply:
D’après Wikipedia, « RFC 6350 also noted that all of the following media types should all be considered deprecated, in favor of text/vcard » (et ceci inclut `text/x-vcard`). En avez-vous tenu compte ?